docs/install_Device_Northbound.md

```diff
@@ -143,165 +141,36 @@ certificates to allow a connection from the IIH.
 
 3. Add two Aspects to structure the data according two our two machines
 
-  ![IIH_CreateAssetModel1](graphics/IIH_CreateAssetModel1.png)
+  ![IIH_CreateAssetModel1](graphics/IIH7.png)
```

Comment:
I would change the Asset Model as the definition for what is an Asset and what is an Aspect is not followed.

Asset is a representation of a physical system like a machine or automation system. (Device, Robot, Production Line, etc.)

Aspects models information for example on a use case basis (e.g. Energy Management) which can be attached to various assets which e.g. have energy consumptions.

Line1 (Asset)
- Media Consumption (Aspect)
  - totalEnergy
  - totalWater
  - ...
Line2 (Asset)
- Media Consumption (Aspect)
  - totalEnergy
  - totalWater
  - ...
  
Also don't name "Line1_TotalEnergy" but used name TotalEnergy as it is already grouped underneath Line1